rdrf/rdrf/static/js/highlight_changes.js

```diff
@@ -45,10 +45,12 @@ $(document).ready(function() {
       var didValueChange = didChange($(this));
 
       var elementToHighlight = $(this);
-      var highlightBorder = true;
-      if ($(this).is(':radio') || $(this).is(':checkbox')) {
+      var highlightBorder = !($(this).is(':radio') || $(this).is(':checkbox'));
```

Comment:
This comment is just an FYI - you don't need to change this.

This selector can be shortened to `$(this).is(":not(:checkbox,:radio)")`
https://api.jquery.com/category/selectors/